_author/axel.md

```diff
@@ -1,6 +1,8 @@
 ---
 fullname: Axel Haustant
 role: Développeur
+start: 2013-07-01
+end: 2014-01-01
```

Comment:
You're right ;)
start: 2013-08-27
end: 2017-02-27
(I'm not sure about the 27, I need to check my contract)